test/runtests.jl

```diff
@@ -477,5 +478,22 @@ using Compat
             @test_broken isready(bottom_dc3) == true
             @test_broken fetch(bottom_dc3) == 14
         end
+
+        @testset "DeferredFuture parallel serialization" begin
+            pnums = addprocs(2)
+            @everywhere using Dispatcher
+
+            try
+                ctx = DispatchContext()
+                exec = ParallelExecutor()
+
+                op = Op(()->3)
+                add!(ctx, op)
+
+                results = run!(exec, ctx)
```

Comment:
Should probably test that this actually returns 3 (vs a wrapped exception or something). Also, I'm a little worried about the DeferredFuture tests depending on dispatcher. Couldn't we just trigger this behaviour by passing a nested `DeferredFuture` between two processes (using `@spawn` directly)?